### PR TITLE
Add additional text regarding issues and FO

### PR DESCRIPTION
**Pull request by pes10k**

### CODE REVIEW

File: howto-conduct-a-privacy-review.md [hunk 1]
```diff
@@ -113,8 +113,13 @@ of a privacy review, or might even file contradictory privacy issues if
 PING members disagree. Disagreement is rare. In the case that a PING privacy review results in 
 contradictory (or mutually exclusive) issues being filed, its the responsibility
 of the working group making the proposal (and, ultimately the director)
-to figure out how to proceed. However, the chairs will work with the working group and reviewer as needed with the aim of resolving any outstanding issues. 
+to figure out how to proceed. However, the chairs will work with the working group and reviewer
+as needed with the aim of resolving any outstanding issues.
 
+If a reviewer is unable to resolve an issue with the working group
+and the issue may lead to a formal objection, the reviewer
+is encouraged to disucss the issue with the PING chairs, and on a PING call.
```
Comment (npdoty) [inline howto-conduct-a-privacy-review.md:121]: I might frame this as (to make it clear that PING will discuss it, not just the chairs):

> the reviewer is encouraged to bring the issue to the PING chairs, who can schedule discussion with PING on a call.

File: howto-conduct-a-privacy-review.md [hunk 2]
```diff
@@ -113,8 +113,13 @@ of a privacy review, or might even file contradictory privacy issues if
...
+PING can then explore other ways to try and resolve the concern without the formal objection.
```
Comment (npdoty) [inline howto-conduct-a-privacy-review.md:122]: An additional goal for PING would be to document the concern and positions of PING reviewers and the Working Group, to facilitate subsequent resolutions of any Formal Objection.

Comment (pes10k) [reply howto-conduct-a-privacy-review.md:122]: thats a good change, thanks! done

**Comment by npdoty** [comment]
Thanks for getting this started, @pes10k. I've suggested a wording change and an addition, but think this will definitely help in any case.

File: howto-conduct-a-privacy-review.md [hunk 3]
```diff
@@ -113,8 +113,13 @@ of a privacy review, or might even file contradictory privacy issues if
...
 contradictory (or mutually exclusive) issues being filed, its the responsibility
 of the working group making the proposal (and, ultimately the director)
-to figure out how to proceed. However, the chairs will work with the working group and reviewer as needed with the aim of resolving any outstanding issues. 
+to figure out how to proceed. However, the chairs will work with the working group and reviewer
+as needed with the aim of resolving any outstanding issues.
 
+If a reviewer is unable to resolve an issue with the working group
+and the issue may lead to a formal objection, the reviewer
+is encouraged to bring the issue to the PING chairs, who can schedule discussion with PING on a call.
+PING can then explore other ways to try and resolve the concern without the formal objection.
```
Comment (npdoty) [inline howto-conduct-a-privacy-review.md:122]:
```suggestion
PING can then explore other ways to try and resolve the concern without the Formal Objection. PING can also  document the concern and positions of PING reviewers and the Working Group, to facilitate subsequent resolutions of any Formal Objection.
```

**Pull request merged**